AmazonChimeSDKDemo/AmazonChimeSDKDemo/Base.lproj/Main.storyboard

```diff
@@ -1108,6 +1237,166 @@
             </objects>
             <point key="canvasLocation" x="169.56521739130437" y="757.36607142857144"/>
         </scene>
+        <!--Root View Controller-->
+        <scene sceneID="Fbo-b3-nFn">
+            <objects>
+                <tableViewController id="lAa-p0-qAc" sceneMemberID="viewController">
+                    <tableView key="view" clipsSubviews="YES" contentMode="scaleToFill" alwaysBounceVertical="YES" dataMode="prototypes" style="plain" separatorStyle="default" rowHeight="-1" estimatedRowHeight="-1" sectionHeaderHeight="-1" estimatedSectionHeaderHeight="-1" sectionFooterHeight="-1" estimatedSectionFooterHeight="-1" id="zte-1A-iEx">
+                        <rect key="frame" x="0.0" y="0.0" width="428" height="926"/>
+                        <autoresizingMask key="autoresizingMask" widthSizable="YES" heightSizable="YES"/>
+                        <color key="backgroundColor" systemColor="systemBackgroundColor"/>
+                        <prototypes>
+                            <tableViewCell clipsSubviews="YES" contentMode="scaleToFill" preservesSuperviewLayoutMargins="YES" selectionStyle="default" indentationWidth="10" id="1yU-vd-a8Z">
+                                <rect key="frame" x="0.0" y="44.666666030883789" width="428" height="43.666667938232422"/>
+                                <autoresizingMask key="autoresizingMask"/>
+                                <tableViewCellContentView key="contentView" opaque="NO" clipsSubviews="YES" multipleTouchEnabled="YES" contentMode="center" preservesSuperviewLayoutMargins="YES" insetsLayoutMarginsFromSafeArea="NO" tableViewCell="1yU-vd-a8Z" id="B6i-fZ-1xG">
+                                    <rect key="frame" x="0.0" y="0.0" width="428" height="43.666667938232422"/>
+                                    <autoresizingMask key="autoresizingMask"/>
+                                </tableViewCellContentView>
+                            </tableViewCell>
+                        </prototypes>
+                        <connections>
+                            <outlet property="dataSource" destination="lAa-p0-qAc" id="Ptu-lv-dVb"/>
+                            <outlet property="delegate" destination="lAa-p0-qAc" id="3Ha-ns-Hpa"/>
+                        </connections>
+                    </tableView>
+                    <navigationItem key="navigationItem" title="Root View Controller" id="YK7-IR-vwY"/>
+                </tableViewController>
+                <placeholder placeholderIdentifier="IBFirstResponder" id="WyL-X2-9Kl" userLabel="First Responder" customClass="UIResponder" sceneMemberID="firstResponder"/>
+            </objects>
+            <point key="canvasLocation" x="-497" y="472"/>
+        </scene>
+        <!--Automatic Language Identification Controller-->
+        <scene sceneID="cFr-V1-DfR">
```

Comment:
Nit: Looks like we created a new controller in storyboard,  storyboard file always makes unrelated changes when we open it, and causing conflicts. In the future we should separate the controllers into dedicated storyboard files to prevent the conflicts.